src/custom-elements/src/components/button/button.tsx

```diff
@@ -18,6 +18,11 @@ export class Button {
    */
   @Prop({ reflect: true }) variant: string;
 
+  /**
+   *  to set data position of a button { outline, flat }.
+   */
+   @Prop({ reflect: true }) dataPosition?: string;
```

Comment:
@soujanreddy for what is needed this prop?

Reply:
@jllr this is to position the dropdown menu for the dropdown button. The current dropdown component is not passing the positions to chi-button.